dts/arm/gigadevice/gd32f4xx/gd32f470.dtsi

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2022, Rtone.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gigadevice/gd32f4xx/gd32f4xx.dtsi>
```

Comment:
Since F470 is F450 with more speed and SRAM, you can just include [gd32f450.dtsi](https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/arm/gigadevice/gd32f4xx/gd32f450.dtsi) here.

Reply:
done.